backend/test/connectors/authorisation/authorisation.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { describe, expect, test, vi } from 'vitest'
+
+import { getAuthorisationConnector } from '../../../src/connectors/v2/authorisation/index.js'
+
+const configMock = vi.hoisted(() => ({
+  connectors: {
+    authorisation: {
+      kind: 'silly',
+    },
+  },
+}))
+vi.mock('../../../src/utils/v2/config.js', () => ({
+  __esModule: true,
+  default: configMock,
+}))
+
+describe('connectors > authorisation', () => {
```

Comment:
Connectors tests - can you add an unhappy test for each connector for the default error case?